num_of_buttons: 16
axis_linear_x: 1
axis_linear_y: 0
axis_linear_z: 3
axis_angular: 2
scale_linear_x: 15.0
scale_linear_y: 15.0
scale_linear_z: 15.0
scale_angular: 15.0
button_speed_up: 12
button_speed_down: 14
button_dead_man: 11
button_arm_dead_man: 10
button_joint_up: 15
button_joint_down: 13
button_ptz_tilt_up: 4
button_ptz_tilt_down: 6
button_ptz_pan_right: 5
button_ptz_pan_left: 7
button_control_mode: 0
tilt_increment: 5
pan_increment: 5
max_angular_position: 0.52
